src/plugins/data_source_management/public/components/utils.ts

```diff
@@ -49,6 +54,29 @@ export async function getDataSourcesWithFields(
   return response?.savedObjects;
 }
 
+export async function handleSetDefaultDatasourceAfterDeletion(
+  savedObjects: SavedObjectsClientContract,
```

Comment:
nit: 
Based on the type "SavedObjectsClientContract", Looks like here we want to provide a savedObjectClient here.
What if we rename ```savedObjects``` to ```savedObjectsClient``` like we we did here: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6237/files#diff-fde383d0fdaaed3bb243ed36abd631c9c311596b2057f737253fc5ab11c4d5edR45?
What do you think?

Reply:
Fixed it!